roundabout-api/src/main/java/at/fhv/itm3/s2/roundabout/api/entity/ICar.java

```diff
@@ -3,6 +3,36 @@
 import java.util.List;
 
 public interface ICar {
+
+    /**
+     * Calculates the time the car needs to traverse the current street section it is standing on
+     *
+     * @return  the traverse time in seconds
+     */
+    double getTimeToTraverseSection();
+
+    /**
+     * Calculates the time the car needs to traverse a given street section
+     *
+     * @param section   the street section we are interested in how long the car needs to traverse it
+     * @return          the traverse time in seconds
+     */
+    double getTimeToTraverseSection(IStreetSection section);
+
+    /**
+     * Calculates the time the car needs until it has moved away from its current spot
+     *
+     * @return  the transition time in seconds
+     */
+    double getTransitionTime();
+
+    /**
+     * Returns the next section the car will be standing on
+     *
+     * @return  the next section as IStreetSection
+     */
+    IStreetSection getNextSection();
+
     double getLastUpdateTime();
```

Comment:
That's not mine code and I don't know how exactly the last update time is defined (what needs to happen that this time is set f.e.), so that should document the developer who has written this code ;)

Reply:
And there are more more methods where java docs are still missing btw (for whoever is responsible for this methods) :P